projects/control-service/projects/pipelines_control_service/src/test/resources/application-unittest.properties

```diff
@@ -25,6 +25,7 @@ datajobs.git.password=
 
 datajobs.control.k8s.namespace=${CONTROL_K8S_NAMESPACE:default}
 datajobs.deployment.k8s.namespace=${DEPLOYMENT_K8S_NAMESPACE:default}
+datajobs.deployment.supportedPythonVersions={3.7: {vdkImage: 'some-test-img', baseImage: 'python:3.7-slim'}}
 
```

Comment:
Missing default version

Reply:
that is acceptable config because it inherits from application.properties where it is set.